crowdsourcing/viewsets/google_drive.py

```diff
@@ -83,6 +85,67 @@ def auth_end(self, request):
 class GoogleDriveViewSet(ViewSet):
     permission_classes = [IsAuthenticated]
 
+    def add_folder(self, request):
+        name = request.data['name']
+        parent = request.data['parent']
+        prev = AccountModel.objects.get(owner=request.user, type='GOOGLEDRIVE').root
+        if parent != "":
+            account = 1
+            drive_util = GoogleDriveUtil(account_instance=account)
+            file_list = drive_util.list_files_in_folder(prev, "blah")
+            for fileobj in file_list:
+                if fileobj['title'] == parent:
+                    prev = fileobj['id']
+                    break
+        account = 1
```

Comment:
what is `account = 1` exactly means? If it's a constant, we can make it as a constant; else we shouldn't hard code it here. We can optionally put it as parameter if needed.